packages/contracts/test/test-utils/dao.ts

```diff
@@ -2,11 +2,20 @@ import {ethers} from 'hardhat';
 import {DAO} from '../../typechain';
 import {deployWithProxy} from './proxy';
 
+export const BYTES32 =
```

Comment:
```suggestion
export const ZERO_BYTES32 =
```

Reply:
Can't it be defined just once, BTW?  already in tests/constants.ts

Reply:
`tests/constants.ts` is in `packages/subgraph` and this definition is in `packages/contracts`.